L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.cc

```diff
@@ -621,6 +621,12 @@ void CSCTriggerPrimitivesBuilder::build(const CSCBadChambers* badChambers,
   // run MPC simulation
   m_muonportcard->loadDigis(oc_lct);
 
+  // temporary hack to ensure that all MPC LCTs are read out
```

Comment:
If these parameters are read from the python configuration or from the conditiosn, why do you need to override them ?
It would be saner to just change the python (or even better the payload) accordingly.